src/protocols/bdx/tests/TestBdxTransferSession.cpp

```diff
@@ -189,14 +184,13 @@ void SendAndVerifyTransferInit(nlTestSuite * inSuite, void * inContext, Transfer
             {
                 // Only check that metadata buffers match. The OutputEvent can still be inspected when this function returns to
                 // parse the metadata and verify that it matches.
-                NL_TEST_ASSERT(
-                    inSuite,
-                    !memcmp(initData.Metadata, outEvent.transferInitData.Metadata, outEvent.transferInitData.MetadataLength));
+                EXPECT_EQ(0,
+                          memcmp(initData.Metadata, outEvent.transferInitData.Metadata, outEvent.transferInitData.MetadataLength));
             }
         }
         else
         {
-            NL_TEST_ASSERT(inSuite, false); // Metadata length mismatch
+            EXPECT_TRUE(false); // Metadata length mismatch
```

Comment:
Refactor test logic to get rid of `ASSERT_TRUE(false)`. Later in the test output such information ("false != true") will be meaningless.

Reply:
Changed in all places that seemed applicable.